labonneboite/web/templates/base.html

```diff
@@ -117,16 +125,19 @@
     {% endif %}
   {% endwith %}
 
-  <div id="content">{% block content %}{% endblock %}</div>
+  <main role="main" id="content">
+    <a id="main-anchor" tabindex="-1"></a>
```

Comment:
When tabindex is set to a negative integer like -1, it becomes programmatically focusable but it isn’t included in the tab order. In other words, it can’t be reached by someone using the tab key to navigate through content, but it can be focused on with scripting.

Reply:
On s'assure ainsi que seule la personne utilisant le lien d'évitement puisse l'atteindre tout en évitant qu'il tombe dessus en parcourant la page au clavier :)